### ISSUE TITLE
Pagination breaks with no Link header

### ISSUE THREAD

**Issue by captn3m0** [chrisanthropic/terraform-import-github-organization#2]
If you have less than 100 public repos, there is no Link header in the response for `get_public_pagination` and the `seq` call breaks as a result:

```
./terraform-import-github-org.sh 
seq: missing operand
Try 'seq --help' for more information.
```

This might be breaking for other pagination calls as well.

**Comment by captn3m0** [comment]
I also see a lot of errors like:

```
./terraform-import-github-org.sh: line 232: github-teams-$TEAM_NAME.tf: ambiguous redirect
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0  542k    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0  542k    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
Error: resource address "github_team_repository.Team-name" does not exist in the configuration.

Before importing this resource, please create its configuration in the root module. For example:

resource "github_team_repository" "Team-name {
  # (resource arguments)
}
```

**Comment by hartzell** [comment]
I've fixed the pagination problem in PR #5 .
